src/InstallationValidator.Core/Assets/Captions.cs

```diff
@@ -33,11 +33,22 @@ public static string ValidationDescription
          }
       }
 
+      
+
+      public static readonly string StopComparison = "Stop Comparison";
+      public static readonly string StartComparison = "Start Comparison";
+      public static readonly string TheComparisonWasCanceled = "The comparison was canceled";
+
+      public static string SimulationComparisonViewTitle = $"{OSPSuite.Core.Domain.Constants.SUITE_NAME} - Simulation Output Comparer";
       public static string MainViewTitle = $"{OSPSuite.Core.Domain.Constants.SUITE_NAME} - Installation Validator";
       public const string New = "New";
       public const string Old = "Old";
       public static readonly string Installation = "Installation";
       public static readonly string Computed = "Computed";
+
+      public static readonly string ComparisonFolder1 = "Comparison Folder 1";
```

Comment:
I think we should avoid imposing the meaning of folder selection. What if you want to compare the results from two different computing platforms? Old and new does not always make sense for this tool.